core/sources/test_helper.move

```diff
@@ -134,14 +141,29 @@ module aptos_names::test_helper {
         // Assert events have been correctly emmitted
         let register_name_event_v1_num_emitted = domains::get_register_name_event_v1_count() - register_name_event_v1_event_count_before;
         let set_name_address_event_v1_num_emitted = domains::get_set_name_address_event_v1_count() - set_name_address_event_v1_event_count_before;
+        let set_reverse_lookup_event_v1_num_emitted = domains::get_set_reverse_lookup_event_v1_count() - set_reverse_lookup_event_v1_event_count_before;
 
         test_utils::print_actual_expected(b"register_name_event_v1_num_emitted: ", register_name_event_v1_num_emitted, 1, false);
         assert!(register_name_event_v1_num_emitted == 1, register_name_event_v1_num_emitted);
 
+        // Reverse lookup should be set if user did not have one before
+        if (option::is_none(&user_reverse_lookup_before)) {
+            assert!(option::is_some(&domains::get_reverse_lookup(user_addr)), 36);
+            assert!(set_reverse_lookup_event_v1_num_emitted == 1, set_reverse_lookup_event_v1_num_emitted);
+        } else {
+            assert!(set_reverse_lookup_event_v1_num_emitted == 0, set_reverse_lookup_event_v1_num_emitted);
+        };
+
         if (is_subdomain) {
-            // We haven't set a target address yet!
-            test_utils::print_actual_expected(b"set_name_address_event_v1_num_emitted: ", set_name_address_event_v1_num_emitted, 0, false);
-            assert!(set_name_address_event_v1_num_emitted == 0, set_name_address_event_v1_num_emitted);
+            if (option::is_none(&user_reverse_lookup_before)) {
+                // Should automatically point to the users address
+                test_utils::print_actual_expected(b"set_name_address_event_v1_num_emitted: ", set_name_address_event_v1_num_emitted, 1, false);
```

Comment:
Should these print statements (print_actual_expected) be removed?

Reply:
I don't think so -- is there something here that makes it unclear?